src/runtime/lpython/lpython.py

```diff
@@ -4,6 +4,8 @@
 import platform
 from dataclasses import dataclass as py_dataclass, is_dataclass as py_is_dataclass
 from goto import with_goto
+from numpy import get_include
```

Comment:
I think we don't want to include numpy here, since it would be imported with every "import lpython" in every file and things will be slow. Python's imports are slow, and we want "import lpython" to be immediate. For now I would "import numpy" when the decorator is processed, not here. This can be done in subsequent PR.